addons/website_mass_mailing/static/src/snippets/s_popup/000.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module **/
+
+import PopupWidget from 'website.s_popup';
+
+PopupWidget.include({
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+	/**
+     * Prevents the (newsletter) popup to be shown if the user is subscribed.
+	 *
+     * @override
+     */
```

Comment:
oops

Reply:
This is getting annoying, I already fixed those:
- [here](https://github.com/odoo/odoo/compare/c0e7730a99ad5a8050cbaef95f03f40f222a9b28..4ef40bcccc1f7afb39e065e220c1380820523d3d)
- [and here](https://github.com/odoo/odoo/compare/4ef40bcccc1f7afb39e065e220c1380820523d3d..03e69900f8c45c76fb185c98540cb4c1666dbb25)

I don't know why/how/since when, but my IDE seems to mix tabs and whitespace (only in js files), so it looks perfectly linted on my end, but not on github/git -_-

Reply:
Done